From: Punit Agrawal <punit@debian.org>
Date: Tue, 5 May 2020 11:01:43 +0900
Subject: libutil/args: Make global variables static

With gcc-10, global fails to compile due to duplicate definition of
"type". Both libutil/argc.c and global/global.c have a global
definition of the variable even though they are completely unrelated.

As the usage in args.c is local to the compilation unit, restrict the
scope of "type" to be file local - fixing the compilation error. While
we are at it, also mark other file local variables as static.

Closes: #957275
---
 libutil/args.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/libutil/args.c b/libutil/args.c
index 142302f..514bedc 100644
--- a/libutil/args.c
+++ b/libutil/args.c
@@ -39,10 +39,10 @@
 #define ARGS_GFIND	3
 #define ARGS_BOTH	4
 
-int type;
-const char **argslist;
-FILE *ip;
-GFIND *gp;
+static int type;
+static const char **argslist;
+static FILE *ip;
+static GFIND *gp;
 
 /**
  * args_open:
